### PR TITLE
PR-1 Create Codebase

### PR DESCRIPTION
**Pull request by artmnzn**
code generated by Gatsby to initiate the project

### CODE REVIEW

File: tailwind.config.js
Comment (diedu89) [inline tailwind.config.js:1]: remove this config file, it should be part of PR-2

File: package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "@mdx-js/react": "^3.0.0",
+    "autoprefixer": "^10.4.16",
+    "gatsby": "^5.12.12",
+    "gatsby-plugin-mdx": "^5.12.3",
+    "gatsby-plugin-postcss": "^6.12.0",
+    "gatsby-source-filesystem": "^5.12.1",
+    "postcss": "^8.4.32",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "tailwindcss": "^3.3.6"
```
Comment (diedu89) [inline package.json:28]: let's remove this and work on it in PR-2

File: src/styles/global.css
Comment (diedu89) [inline src/styles/global.css:1]: part of PR-2

File: postcss.config.js
Comment (diedu89) [inline postcss.config.js:1]: same, another tailwind config

File: gatsby-browser.js
Comment (diedu89) [inline gatsby-browser.js:1]: also tailwind related

**Review by diedu89 — CHANGES_REQUESTED** [review]
We're missing eslint and prettier config

**Comment by artmnzn** [comment]
@diedu89 When created codebase Gatsby allowed me to choose tailwind style engine

**Review by diedu89 — APPROVED** [review]
never mind LGTM if gatsby was able to add it within the defaults

**Comment by diedu89** [comment]
sorry I forgot we're still missing eslint and prettier config

**Review by diedu89 — APPROVED** [review]
eslint and prettier will be configured on other ticket

**Pull request merged**